demos/button.html

```diff
@@ -20,10 +20,10 @@
     <title>Button - Material Compoonents Catalog</title>
     <meta name="viewport" content="width=device-width, initial-scale=1">
     <link rel="icon" type="image/png" href="/images/logo_components_color_2x_web_48dp.png">
-    <script src="/assets/demo-styles.css.js"></script>
     <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto+Mono">
     <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,400,500">
     <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
+    <script src="/assets/demo-styles.css.js"></script>
```

Comment:
I'm not sure why this line was moved in this PR, but it will no longer exist the next time you merge with master anyway. The styles have been split out, so now the ones for this demo are in `button.scss`.

Reply:
I will rebase this PR on #1285 .

To address your question though, this line introduced a funny bug. We defined `mdc-button__icon` in `button.scss`, which is sourced as part of `material-component-web.css` in `demo.scss`. However, we added "Material+Icons" stylesheet below that to introduce `material-icon`. The latter introduced a `font-size: 24px` that override what we provided in `demo.scss`(`button.scss`) and that's the reason why I moved it to the bottom because we definitely don't want a general library to override our own styles.

It shouldn't be a concern any longer after I rebased it.